src/vs/workbench/services/languageRuntime/common/languageRuntimeUiClient.ts

```diff
@@ -85,6 +85,7 @@ export class UiClientInstance extends Disposable {
 	onDidWorkingDirectory: Event<WorkingDirectoryEvent>;
 	onDidShowUrl: Event<ShowUrlEvent>;
 	onDidShowHtmlFile: Event<IShowHtmlUriEvent>;
+	onDidHolovizExtensionLoad: Event<HolovizExtensionLoadEvent>;
```

Comment:
Super nit: I think the pattern is typically `onDid<Verb><Object>` so maybe `onDidLoadHolovizExtension`?

Reply:
That flows better as well!